### PR TITLE
Add Conjure-up docs

### PR DESCRIPTION
**Pull request by WillMoggridge**
Add Conjure-up docs from https://github.com/CanonicalLtd/docs-conjure-up/

## QA
 - ./rebuild-docs
 - ./run
 - Go to http://localhost:8007/ and check the Conjure-up section and click through
 - You should end up at http://localhost:8007/conjure-up/en
 - Click around a bit, make sure things look pretty

### CODE REVIEW

**Comment by WillMoggridge** [comment]
[Release conjure-up.io docs](https://trello.com/c/7jxPLY0d/2847-release-conjure-up-io-docs)

File: templates/index.html [hunk 1]
```diff
@@ -43,6 +43,13 @@
                   </td>
               </tr>
               <tr>
+                  <td><a href="https://conjure-up.io" class="p-link--external">Conjure-up</a></td>
+                  <td>
+                      <p>Documentation for Conjure-up.</p>
+                      <p><a href="/conjure-up/en" class="p-link--external">Read the Conjure-up documentation</a></p>
```
Comment (anthonydillon) [inline templates/index.html]: Not an external link.

**Comment by nottrobin** [comment]
@anthonydillon you need to install the `repo` command. This needs adding to the README (#43). I'll try to do that now.

**Comment by nottrobin** [comment]
I've added README instructions for installing the repo command: https://github.com/canonical-websites/docs.ubuntu.com/pull/54

**Comment by nottrobin** [comment]
@anthonydillon fixed that issue - would you mind giving it your stamp of awesome?

**Pull request merged**